src/styles/blocks/_page.scss

```diff
@@ -0,0 +1,74 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #3e3e3e;
+  font-size: 16px;
+  line-height: 27px;
+  scroll-behavior: smooth;
+  
+  &__body {
+    background: $c-gray;
```

Comment:
Background color is a bit off, you should change it according to figma

Reply:
$c-gray: #f2f2f2;
I copied it from figma. I have selected background and it show this color. Maybe its wrong? could you recommend me the right one?
https://prnt.sc/1txx2pd

Reply:
Yeah sure, #f2f2f2 is background color of the whole figma sheet, the actual color of page is just #ffffff (just white)

https://user-images.githubusercontent.com/37161315/135085814-cb04d8a0-7c2d-40e9-9f2e-53fc30a07dce.mov

Reply:
ok) done